packages/did-ipfs-store/src/didStore.ts

```diff
@@ -48,6 +67,36 @@ export class DidStore implements IDidStore {
     }
   }
 
+  /**
+   * Checks if file is pinned on cluster
+   * @param cid CID
+   * @param isFullyReplicated specifies whether to check if data is fully replicated or pinned at least on one node
```

Comment:
- Is there a different in meaning between "pinned" and "replicated" in the IPFS context? Maybe there is, so best to be precise and use "pinned" rather than "replicated"
- Seems to check REPLICATION_FACTOR_MIN rather than "one node"?
```suggestion
   * @param isFullyReplicated specifies whether to check if data is pinned to all nodes in the cluster or pinned to at least REPLICATION_FACTOR_MIN nodes
```